public/app/percona/settings/Settings.service.ts

```diff
@@ -26,12 +26,36 @@ export const SettingsService = {
     return response;
   },
   async setSettings(body: any, setLoading: LoadingCallback) {
-    let response: any;
+    let response: Settings = {
```

Comment:
Should this be `const` now?

Reply:
The var might be reassigned when the API responds :S